crates/dap/src/adapters.rs

```diff
@@ -203,39 +209,7 @@ pub trait DebugAdapter: 'static + Send + Sync {
         &self,
         delegate: &dyn DapDelegate,
         config: &DebugAdapterConfig,
-        adapter_path: Option<PathBuf>,
```

Comment:
`adapter_path` shouldn't be removed because it's when a user want's to use an already installed adapter on their system. 

For example, a debugpy developer could have two versions of debugpy that they want to test and they're be able to by manually inputting the path